### PR TITLE
Add TK - TECNOKENA

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN - URL doesn't resolve. I'll send an email questioning, but we should go ahead and add it (I assume they will have https)

### CODE REVIEW

**Review by palemieux — CHANGES_REQUESTED** [review]
URL does not resolve

File: src/main/data/studios.json [hunk 1]
```diff
@@ -1300,6 +1300,16 @@
...
   },
+  {
+    "code": "TK",
+    "contact": {
+      "address": "R. Dr. Roberto Barrozo, 1778 - Curitiba - PR - BRAZIL - CEP 80810090",
+      "email": "paulo@tecnokena.com.br",
+      "name": "Mr. Paulo Munhoz"
+    },
+    "description": "TECNOKENA",
+    "url": "https://www.tecnokena.com.br"
```
Comment (palemieux) [inline src/main/data/studios.json:1311]:
```suggestion
    "url": "http://www.tecnokena.com.br"
```

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by palemieux** [comment]
@japierce1028 It looks like `https` does not work. I suggest changing it to `http`, ok?

**Comment by japierce1028** [comment]
Seems right to me...

**Comment by japierce1028** [comment]
Looks like I can't commit. Only close.

**Review by palemieux — APPROVED** [review]

**Pull request merged**